.gitignore

```diff
@@ -45,3 +45,6 @@ app.*.map.json
 /android/app/debug
 /android/app/profile
 /android/app/release
+
+# File to keep Customer.IO keys private
+/assets/secrets/keys.json
```

Comment:
Can't we have an `env` file?

Reply:
Yeah, we can rename it to `env`, will update it.

Reply:
I just saw [flutter_dotenv](https://pub.dev/packages/flutter_dotenv) that you used, will replace it in next PR.